prow/cmd/deck/template/base.html

```diff
@@ -6,6 +6,14 @@
   {{if googleAnalytics}}
   <!-- Global site tag (gtag.js) - Google Analytics -->
   <script async src="https://www.googletagmanager.com/gtag/js?id={{googleAnalytics}}"></script>
+  <script>
```

Comment:
This is pulling in an older version of Google Analytics on top of the one we already have - we don't want to include Google Analytics twice.